Tests/SnapshotTests/Extensions/UIColor+DesignSystem.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIColor+DesignSystem.swift
+//  BackbaseDesignSystemSnapshotTests
+//
+//  Created by Backbase R&D B.V. on 09/02/2023.
+//  Copyright © 2023 Backbase R&D B.V. All rights reserved.
+//
+
+import UIKit
+import BackbaseDesignSystem
+
+extension UIColor {
+
+    static var designSystem: DesignSystem.Colors {
```

Comment:
are these extensions used anywhere? if not we should remove them, but if this is used i would rename this as its misleading,  should maybe be `designSystemColors`